### ISSUE TITLE
Start on initial function stubs

### ISSUE THREAD

**Issue by hmacdope** [MDAnalysis/solvation-analysis#5]
Start to lay out the most basic of functions/classes you want to implement, along with some PyTest tests!

Two kinds of tests I think we should possibly do:

- the numpy array kind on some data we can reason about easily
- a test on some realistic data that you know well (you may need to upload some).

**Comment by orioncohen** [comment]
For item 1, are there any prebuilt test trajectories that are just regular grids of atoms? That seems like the ideal test bed for very simple tests.

For item 2, I have uploaded a trajectory that I am pretty familiar with. I'll start writing tests on that.

**Comment by hmacdope** [comment]
Heyo!

This is an example of how to make a trajectory with evenly spaced atoms, you should be able to adapt it to whatever purpose. :)

```python
n_grid = 5
n_frames = 10
n_particles = n_grid**3 #3D grid

grid = np.mgrid[0:n_grid,0:n_grid,0:n_grid] # make the grid
frame = grid.reshape([3,n_particles]).T # make it the right shape

traj = np.empty([n_frames, n_particles, 3])
for i in range(n_frames):
	traj[i,:,:] = frame  # copy the coordinates to 10 frames

u = mda.Universe.empty(n_particles) # jam it into a universe
u.load_new(traj)
# voila

```

**Comment by orioncohen** [comment]
Awesome! Thanks Hugo!

**Comment by orioncohen** [comment]
This is mostly addressed by the cursed PR #6. Should I close this now that there are function stubs and base functionality?

**Comment by hmacdope** [comment]
Yep sounds good. Lets raise some more issues as they arise. I will add some now.
